ocdata/combined.py

```diff
@@ -587,3 +601,22 @@ def get_adsorbed_bulk_dict(self, ind):
             + "_"
             + ads_sampling_str,
         }
+
+    def set_center_of_mass(self, atoms, com):
```

Comment:
Do we no longer need to set center of mass now that we're using binding atom? If so, this method is now unused.

Reply:
It's unused here, but we'll be adding support for it later when we introduce the full random configurations.